docs/tutorials/upgrading-airbyte.md

```diff
@@ -0,0 +1,89 @@
+_Coming soon! This tutorial is a work in progress for a feature that is currently in development. It is not usable yet!_
+
+# Upgrading Airbyte
+
+## Overview
+
+Airbyte releases a new version of its stable version at least once a week. Some of these releases require updating the data the Airbyte stores internally. This tutorial will describe how to determine if you need to run this ugprade process, and if you do, how to do so. This process does require temporarily turning off Airbyte.
+
+## Determining if you need to Upgrade
+Not every new version will require an upgrade. We follow standard [Semantic Versioning](https://semver.org/) conventions. Data migrations should only be required when upgrading to a new major version of Airbyte. You can always find the latest stable version of Airbyte in our repository [here](https://github.com/airbytehq/airbyte/blob/master/.env#L1). If you are upgrading to a new major version follow the steps below.
+
+If you inadvertently upgrade to a version of Airbyte that is not compatible with your data, the docker containers will not start up and will log an error stating the incompatibility. In these cases, you should downgrade to the previous version that worked and follow the steps below.
```

Comment:
not true. currently we will happily corrupt your data. issue: https://github.com/airbytehq/airbyte/issues/1639

Reply:
Looks like a a high priority item.

Reply:
agreed.